content/docs/2.0/concepts/scaling-deployments.md

```diff
@@ -93,6 +103,33 @@ Minimum number of replicas KEDA will scale the deployment down to. By default it
 
 This setting is passed to the HPA definition that KEDA will create for a given deployment.
 
+---
+
+```yaml
+advanced:
+  horizontalPodAutoscalerConfig:
+    resourceMetrics:
+    - name: cpu/memory
+      target:
+        type: value/ utilization/ averagevalue
+        value: 60 # Optional
+        averageValue: 40 # Optional
+        averageUtilization: 50 # Optional
+    behavior: 
+```
+advanced:
```

Comment:
Maybe it's better to follow the approach from the guidance above where it's more of a fluent text rather than annotated YAML?

Reply:
I made the changes and pushed. But github is not reflecting that commit and changes. Figuring out why

Reply:
They are having some service issues, maybe it's related

Reply:
now its shown :)

Reply:
@tomkerkhove The new line doesn't get shown in the rich view. Could you help how I can make it look exactly like the pr?